test/core/extensions/coreIssuanceOrder.spec.ts

```diff
@@ -37,24 +40,23 @@ const { SetProtocolTestUtils: SetTestUtils, SetProtocolUtils: SetUtils } = setPr
 const setTestUtils = new SetTestUtils(web3);
 const setUtils = new SetUtils(web3);
 const { expect } = chai;
-const { NULL_ADDRESS } = SetUtils.CONSTANTS;
+const { NULL_ADDRESS, ZERO } = SetUtils.CONSTANTS;
 
 
 contract('CoreIssuanceOrder', accounts => {
   const [
     contractDeployerAccount,
-    takerAccount,
-    makerAccount,
-    signerAccount,
+    issuanceOrderTaker,
+    issuanceOrderMaker,
     relayerAccount,
-    zeroExOrderMakerAccount,
+    zeroExOrderMaker,
+    notIssuanceOrderMakerAccount,
```

Comment:
Do we want to phase out the suffix of `Account` completely?

Reply:
Yeah, +1 i'll update for consistency